.kokoro/nightly/samples.cfg

```diff
@@ -22,6 +22,11 @@ env_vars: {
   value: "java-docs-samples-testing"
 }
 
+env_vars: {
+  key: "DIALOGFLOW_CX_AGENT_ID"
+  value: "b8d0e85d-0741-4e6d-a66a-3671184b7b93"
+}
+
```

Comment:
This file is templated and will be overwritten by automation unless we add to the list of excludes in the root `synth.py` file. Additionally, these env variables will require adding to the allowlist in the internal Kokoro job configuration.

I suggest we remove this configuration and set the default in the tests which can be overridden by the environment variable.

Reply:
Got it, thank you!